schemas/README.md

```diff
@@ -2335,7 +2333,52 @@ bytes
 </td>
 <td>
 
-Raw image data
+Raw image data.
+
+The requirements for each `encoding` value are:
+- `yuv422`,  `uyvy`:
+  - 8-bit [Y'UV](https://en.wikipedia.org/wiki/Y%E2%80%B2UV).
+  - U and V values are shared between horizontal pairs of pixels. Each pair of output pixels is encoded as [U, Y1, V, Y2].
+  - `step` must be greater than or equal to `width` * 2.
+- `yuv422_yuy2`,  `yuyv`:
+  - 8-Bit [Y'UV](https://en.wikipedia.org/wiki/Y%E2%80%B2UV).
+  - U and V values are shared between horizontal pairs of pixels. Each pair of output pixels is encoded as [Y1, U, Y2, V].
+  - `step` must be greater than or equal to `width` * 2.
+- `rgb8`:
+  - 8-bit RGB.
+  - Each output pixel is encoded as [R, G, B].
+  - `step` must be greater than or equal to `width` * 3.
+- `rgba8`:
+  - 8-bit RGB + Alpha.
+  - Each output pixel is encoded as [R, G, B, Alpha].
+  - `step` must be greater than or equal to `width` * 4.
+- `bgr8`:
+  - RGB, one byte per component.
```

Comment:
RGB is backwards here relative to `bgr8`, what does that mean? Also why does this say "one byte per component" whereas the other ones are `8-bit`, is there some difference?